media/css/firefox/challenge-the-default/_animations.scss

```diff
@@ -0,0 +1,121 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$campaign-red: #ff6a75;
+$campaign-pink: #ffbdc5;
+$campaign-mint: #d1ffee;
+$campaign-green: #2ac3a2;
+
+.js {
+    @media (prefers-reduced-motion: no-preference) {
+
+        @keyframes pop-in {
+            0% {
+                opacity: 0;
+                transform: scale(0.3);
+            }
+
+            35% {
+                opacity: 1;
+                transform: scale(0.3);
+            }
+
+            50% {
+                transform: scale(0.7);
+            }
+
+            65% {
+                transform: scale(0.7);
+            }
+
+            80% {
+                transform: scale(1.1);
+            }
+
+            90% {
+                transform: scale(1.1);
+            }
+
+            100% {
+                opacity: 1;
+                transform: scale(1);
+            }
+        }
```

Comment:
**Suggestion:** The pop-in animation stutters a little and sort of creates a dizzying affect. I think a big reason is because of the scales we have set here. I played around with the measurements, and what I came up with seems _plenty_ smoother. It also really improves how the `picto` elements are showing up, too.
Give it a shot and let me know what you think:

```
@keyframes pop-in {
            0% {
                opacity: 0;
                transform: scale(0);
            }

            35% {
                opacity: 1;
                transform: scale(0.2);
            }

            50% {
                transform: scale(0.5);
            }

            65% {
                transform: scale(0.7);
            }

            80% {
                transform: scale(0.9);
            }

            90% {
                transform: scale(1.1);
            }

            100% {
                opacity: 1;
                transform: scale(1);
            }
        }
```

Reply:
Good suggestion! This was the first thing I noticed when the page loaded to. Your code snippet definitely seems to improve things for me as well.

Reply:
Nice idea, thinks it works real well.